lib/openrtb2_3/deal.js

```diff
@@ -0,0 +1,76 @@
+'use strict'
+
+const RtbObject = require('../rtbObject');
+const removeEmptyValues = require('../utils').removeEmptyValues;
+const dealSchema = require('./schemas/deal');
+const ValidationError = require('../errors/ValidationError');
+const validator = require('../validator');
+
+var Deal = function(id, bidfloor, bidfloorcur, at, wseat, wadomain, ext) {
+  this.id = id;
+  this.bidfloor = bidfloor;
+  this.bidfloorcur = bidfloorcur;
```

Comment:
[LOW] we should add an example of bidfloorcur in the readme as you did with the other properties